DragonFruit.Six.API/Data/Extensions/AccountInfoExtensions.cs

```diff
@@ -27,5 +27,17 @@ public static class AccountInfoExtensions
             var request = new AccountInfoRequest(platform, lookupMethod, queries);
             return client.Perform<JObject>(request, token).DeserializeAccountInfo();
         }
+
+        /// <summary>
+        /// Get a user's account info by name
+        /// </summary>
+        public static AccountInfo GetUserByName<T>(this T client, string name, Platform platform) where T : Dragon6Client
+            => client.GetUser(platform, LookupMethod.Name, name);
+
+        /// <summary>
+        /// Get a user's account info by userId
```

Comment:
same as above, worth XMLDocing that this is for a ubisoft id, but then it says that in the method name...